lib/compilers/dotnet.ts

```diff
@@ -72,46 +72,13 @@ class DotNetCompiler extends BaseCompiler {
         const programDir = path.dirname(inputFileName);
         const sourceFile = path.basename(inputFileName);
 
-        const projectFilePath = path.join(programDir, `CompilerExplorer${this.lang.extensions[0]}proj`);
-        const crossgen2Path = path.join(this.clrBuildDir, 'crossgen2', 'crossgen2.dll');
-
-        const programPublishPath = path.join(
-            programDir,
-            'bin',
-            this.buildConfig,
-            this.targetFramework,
-            this.rID,
-            'publish',
-        );
-
-        const programDllPath = path.join(programPublishPath, 'CompilerExplorer.dll');
-        const projectFileContent =
-            `<Project Sdk="Microsoft.NET.Sdk">
-            <PropertyGroup>
-                <TargetFramework>${this.targetFramework}</TargetFramework>
-                <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-                <Nullable>enable</Nullable>
-                <AssemblyName>CompilerExplorer</AssemblyName>
-                <LangVersion>${this.langVersion}</LangVersion>
-                <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
-                <EnablePreviewFeatures>${this.langVersion === 'preview' ? 'true' : 'false'}</EnablePreviewFeatures>
-                <RestoreAdditionalProjectSources>
-                  https://api.nuget.org/v3/index.json;${this.additionalSources ? this.additionalSources : ''}
-                </RestoreAdditionalProjectSources>
-            </PropertyGroup>
-            <ItemGroup>
-                <Compile Include="${sourceFile}" />
-            </ItemGroup>
-         </Project>
-        `;
-
+        await fs.copy(this.testAppSrc, programDir);
+        
+        execOptions.env.DOTNET_TC_QuickJitForLoops = 'true';
```

Comment:
Doesn't QJFL decrease the quality of jitted code?

Reply:
No, it doesn't affect crossgen's output codegen but it makes it slightly faster (both msbuild and crossgen). On my machine I managed to shave 0.5s with it.